jekyll/_cci2/container-runner.adoc

```diff
@@ -390,6 +390,8 @@ NOTE: Cluster-wide permissions are used by container runner to autodetect the OS
 
 Each container runner has a garbage collector which will ensure any pods and secrets with the label `app.kubernetes.io/managed-by=circleci-container-agent` left dangling in the cluster are removed. By default this will remove all jobs older than five hours and five minutes. This can be shortened or lengthened via the `agent.kubeGCThreshold` parameter. However, if you do shorten the garbage collection (GC) frequency, also shorten the max task run time via the `agent.maxRunTime` parameter to be a value smaller than the new GC frequency. Otherwise a running task pod could be removed by the GC.
```

Comment:
~ `agent.kubeGCThreshold` is a deprecated configuration parameter for the GC. We should reference these values: https://github.com/CircleCI-Public/container-runner-helm-chart/blob/af248a32f37fa2663356d7134d78a2dea6362053/values.yaml#L161-L175